tests/test_fixtures.py

```diff
@@ -332,3 +332,15 @@ def test_unblock_manually(self, django_db_blocker):
     def test_unblock_with_block(self, django_db_blocker):
         with django_db_blocker.unblock():
             Item.objects.exists()
+
+
+@pytest.mark.parametrize('run_no', [(1,), (2,)])
+def test_clear_site_cache(run_no):
+    """
+    Run this two times, would fail on run #2 if SITE_CAHCE would not be cleared.
+
+    Note that this test might start to fail if we choose to use pytest-xdist -n.
+    """
+    from django.contrib.sites.models import SITE_CACHE
+    assert len(SITE_CACHE) == 0
+    SITE_CACHE[run_no] = 'run_no {}'.format(run_no)
```

Comment:
From what I understand (see the comment about the `SITE_CACHE.clear()` too): SITE_CACHE is not part of a public Django API. This test should be rewritten to make sure the cache is cleared by checking `Site.objects.get_current()` which seems to be the public API to interact with the site framework.

Something like this will probably do (I have not actually run this code and probably missed some detail):
```py
@pytest.mark.django_db
@pytest.mark.parametrize('name', ['site1', 'site2'])
def test_clear_site_cache(settings, site_name):
    settings.SITE_ID = 1
    Site.objects.create(id=1, domain='foo.com', name=site_name)
    assert Site.objects.get_current().name == site_name
```